web_external/js/views/body/ImagesView/ImageDetailsPane.js

```diff
@@ -1,46 +1,66 @@
 isic.views.ImagesViewSubViews = isic.views.ImagesViewSubViews || {};
 
 // View for image details
-isic.views.ImagesViewSubViews.ImageDetailsPane = Backbone.View.extend({
+isic.views.ImagesViewSubViews.ImageDetailsPane = isic.View.extend({
     events: {
         'click .button': 'clearSelectedImage'
     },
+
     initialize: function () {
         this.image = new isic.models.ImageModel();
-        this.listenTo(this.image, 'change', this.render);
-        this.listenTo(this.model, 'change:selectedImageId', this.fetchImage);
+        this.listenTo(this.image, 'g:fetched', this.render);
+        this.listenTo(this.model, 'change:selectedImageId', this.selectedImageChanged);
+
+        this.segmentationsDisplayView = new isic.views.SegmentationsDisplayView({
+            image: this.image,
+            parentView: this
+        });
     },
 
     render: function () {
-        // Get metadata from image
+        var created = null;
         var acquisitionMetadata = null;
         var clinicalMetadata = null;
-        if (this.image.has('meta')) {
+
+        // Get image data
+        if (this.image.id) {
+            created = girder.formatDate(this.image.get('created'), girder.DATE_SECOND);
             var meta = this.image.get('meta');
-            if (_.has(meta, 'acquisition')) {
-                acquisitionMetadata = meta['acquisition'];
-            }
-            if (_.has(meta, 'clinical')) {
-                clinicalMetadata = meta['clinical'];
+            acquisitionMetadata = meta['acquisition'];
+            clinicalMetadata = meta['clinical'];
+
+            // Reformat some acquisition metadata
+            if (_.has(acquisitionMetadata, 'pixelsX') &&
```

Comment:
`pixelsX` and `pixelsY` will always exist: https://github.com/ImageMarkup/isic-archive/blob/master/server/models/image.py#L93